design/eti/override/templates/full/dichiarazioni.tpl

```diff
@@ -42,7 +42,7 @@
     'plugins/chosen.jquery.js',
     'moment.min.js',
     'jquery.dataTables.js',
-    'jquery.opendataDataTable.js',
+    'jquery.patEtiDataTable.js',
```

Comment:
Facciamo un plign dedicato per poter isolare l'estensione anche in questo javascript